terraform/module-magento/variables.tf

```diff
@@ -156,7 +153,7 @@ variable "elasticache_engine" {
 }
 
 variable "elasticache_parameter_group_name" {
-  default = "default.redis3.2"
+  default = "default.redis4.0"
```

Comment:
Magento doesn't support `Redis 4.0` officially https://devdocs.magento.com/guides/v2.3/cloud/project/project-conf-files_services-redis.html

Reply:
nice catch, I fixed it back to 3.2